### DISCUSSION TITLE
Expect a fluentd-plugin-rocketmq like fluentd-plugin-kafka

### DISCUSSION THREAD

**Discussion by huangwei2013** [fluent/fluentd#4773] [Ideas]
### Is your feature request related to a problem? Please describe.

Expect a fluentd-plugin-rocketmq like fluentd-plugin-kafka

### Describe the solution you'd like

Expect a fluentd-plugin-rocketmq like fluentd-plugin-kafka

### Describe alternatives you've considered

Expect a fluentd-plugin-rocketmq like fluentd-plugin-kafka

**Comment by daipom** [comment]
https://github.com/apache/rocketmq
https://github.com/fluent/fluent-plugin-kafka
